nav2_map_server/test/unit/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 include_directories(${PROJECT_SOURCE_DIR}/test)
 
-ament_add_gtest(test_occ_grid test_occ_grid.cpp ${PROJECT_SOURCE_DIR}/test/test_constants.cpp)
```

Comment:
> Were these tests in this file rolled into another file? 

Yes, these tests were moved into `unit/test_map_io.cpp` because they no more covering OccGridLoader functionality.